code/modules/antagonists/_common/antag_spawner.dm

```diff
@@ -239,8 +239,8 @@
 	icon = 'icons/obj/wizard.dmi'
 	icon_state = "vial"
 
-	var/shatter_msg = "<span class='notice'>You shatter the bottle, no turning back now!</span>"
-	var/veil_msg = "<span class='warning'>You sense a dark presence lurking just beyond the veil...</span>"
```

Comment:
i dont like this, people might want to change their spans, why are you limiting them at that
spans are macros anyway just put macros in the var definition